### ISSUE TITLE
Unsupported ROW_NUMBER() OVER () syntax

### ISSUE THREAD

**Issue by morgothulhu** [oguimbal/pgsql-ast-parser#21]
Hello @oguimbal 

Another case not supported yet - thanks for looking!

## Query part not supported
`SELECT ROW_NUMBER() OVER (ORDER BY RES."createdAt" DESC) AS row_number`

## Error message
```
with Error: Syntax error at line 6 col 25:

        ROW_NUMBER() OVER (
                          ^
Unexpected lparen token: "(". Instead, I was expecting to see one of the following:

    - A "comma" token
    - A "kw_from" token
    - A "kw_where" token
    - A "kw_group" token
    - A "kw_order" token
    - A "kw_limit" token
    - A "kw_offset" token
    - A "kw_fetch" token
    - A "kw_union" token
    - A "rparen" token
```

## Raw query
```
SELECT
  * FROM (
    SELECT
      *,
      ROW_NUMBER() OVER (ORDER BY RES."createdAt" DESC) AS row_number
    FROM (
      SELECT DISTINCT ON (GS."GC")
    GS."createdAt" AS "createdAt",
    GC.*
    FROM schema."GaSe" GS
      INNER JOIN schema."GC" GC ON GC."id" = GS."GC"
      WHERE
        GS."owner"='ABC' AND GS."status"='Completed'
      ORDER BY GS."GC", GS."createdAt" DESC
    ) RES
  ) ORES
ORDER BY row_number ASC
LIMIT 20;
```

**Comment by eddieajau** [comment]
And there's another use case:

```
select case_status
  , count(*)::int as count
  , (count(*) / (sum(count(*)) over() )) as ratio
from vw_nba_success
```

**Comment by eddieajau** [comment]
pgsql-ast-parser@6.3.2

SQL:

```
select case_status
  , count(*)::int as count
  , (count(*) / (sum(count(*)) over() )) as ratio
from vw_nba_success
```

Error:

```
Error Error: Syntax error at line 3 col 37:

    , (count(*) / (sum(count(*)) over (
                                      ^
Unexpected lparen token: "(". I did not expect any more input. Here is the state of my parse table:

    kw_between → %word ● 
    kw_operator → %word ● 

    at Parser.feed (/node_modules/nearley/lib/nearley.js:343:27)
    at _parse (/node_modules/pgsql-ast-parser/index.js:1752:16)
    at doParse (/node_modules/pgsql-ast-parser/index.js:1708:27)
    at Object.parse (/node_modules/pgsql-ast-parser/index.js:1710:11)
```

**Comment by oguimbal** [comment]
Hi ! Thanks for the report.
FYI I've started implementing this (I'm almost done), but I lack time these days.
I may find some this week, hopefully.

**Comment by oguimbal** [comment]
Should be ok with `pgsql-ast-parser@7.0.0` 🎉

**Comment by oguimbal** [comment]
Oops, I did not include `OVER()` in this release. Reopening

**Comment by oguimbal** [comment]
Fixed in `pgsql-ast-parser@7.0.1` ... for good this time :)
